src/lib/learn-client/api/api-types.ts

```diff
@@ -75,6 +76,24 @@ export interface ApiCollectionTutorial
 	tutorial: ApiTutorial
 }
 
+export interface ApiCollectionTutorialProgress
+	extends Pick<BaseModel, 'id' | 'created_at' | 'updated_at'> {
+	user_id: string
+	collection_id: uuid
+	tutorial_id: uuid
+	complete_percent: TutorialProgressPercent
+	/**
+	 * Note: so far, we've only needed the tutorial_id from this record type.
+	 * Perhaps we could not include detailed tutorial data in the API response?
+	 */
+	tutorial: ApiTutorialLite
```

Comment:
Added notes here as I _think_ we'll be able to drop these from the API response and significantly reduce the amount of data we're fetching. But very much outside the scope of this PR, and also I'm not _sure_ we'd want to drop the data since I don't have full context on why it was added.

Reply:
We definitely should hang onto these until the API response does otherwise. 😊

Reply:
For sure! 💯  Intent here was to make a note of this in case anyone wonders why these are included in the payload - as far as I can tell, we don't have a strong reason to include them.

Reply:
I don't think we would update the API to not return this data so that the endpoint aligns with other enrichment patterns in the API but we could remove returning this in the client methods during formatting if its not needed in the application.